src/clients/host-client.js

```diff
@@ -47,7 +48,7 @@ class HostClient extends BaseEvernodeClient {
     }
 
     async isRegistered() {
-        return (await this.getRegistration()) !== undefined
+        return (await this.getRegistrationNft()) !== null;
```

Comment:
Shall we add a TODO comment like
**[TODO] This is a temporary fix until getRegistration get fixed.**